rails/config/routes.rb

```diff
@@ -545,6 +545,12 @@
 
     get '/resources/:type/:id_or_filter_value(/:slug)' => 'home#stem_resources', :as => :redirect_stem_resources
 
+    # Custom project page. This route should be always at the very bottom,
+    # so the custom page URL can't overwrite another resource URL!
```

Comment:
Seems like the comment should be updated since it's no longer at the very bottom.

Reply:
@dougmartin can you update the comment and explain why it has to be above the catch all?
Hopefully we can move toward removing the catch all, so I'm not too worried about a project page overriding something picked up by the catch all.